src/core/utils.js

```diff
@@ -751,19 +751,17 @@ export function getLinkTargets(elem) {
 export function renameElement(elem, newName) {
   if (elem.localName === newName) return elem;
   const newElement = elem.ownerDocument.createElement(newName);
-
+  if (!elem.isConnected) throw new Error("Node must be attached to document.");
```

Comment:
This seems to be not supported in Edge yet.
![image](https://user-images.githubusercontent.com/8426945/46160910-a6dfad00-c2a1-11e8-8787-f7d322f12255.png)

Reply:
Can probably do `document.contains(elem)`

Reply:
good suggestion!